core/web/presenters/jsonapi.go

```diff
@@ -14,6 +15,11 @@ func NewJAID(id string) JAID {
 	return JAID{id}
 }
 
+// NewPrefixedJAID prefixes JAID with chain id in %s/%s format.
+func NewPrefixedJAID(id string, chainID string) JAID {
+	return JAID{ID: fmt.Sprintf("%s/%s", chainID, id)}
```

Comment:
The flipped order is surprising. Would it be a pain to change at the call sites? (maybe an IDE can do it for you?)